### ISSUE TITLE
Native HA integration

### ISSUE THREAD

**Issue by CFenner** [lkempf/casambi-bt-hass#52]
Do you plan to contribute this integration into home assistant at one point in time?

**Comment by lkempf** [comment]
Yes. But it's a combination of when I feel that the integration is mature enough (I think it's slowly getting there) and when I think I'll have plenty of time. I'm still having issues with some hard to catch race conditions in the bluetooth connection code so I'll wait at least until I stop finding new ones every time I stop to think about it.

**Comment by CFenner** [comment]
So how is the state of the integration now 😉 shall we try to integrate it? Or maybe first go with https://hacs.xyz/docs/publish/include/

**Comment by lkempf** [comment]
I've seen how much work it is and how frustrating it seems to add significantly simpler integrations to HA and I don't think I have the motivation for that (but would try to help if someone were to try it). But good idea to add it to HACS by default. Setting up one more GitHub action seems a lot more doable.
